report/src/App.tsx

```diff
@@ -116,14 +122,22 @@ const App = () => {
               .split("/")
               .map((i) => parseInt(i as string));
 
-            const total = supported + mismatch + stub;
-            const percentage = pct(total, baselineSupport as number);
+            const totalImplemeneted = supported + mismatch + stub;
```

Comment:
I suggest we move away from "implemented" or "supported" and instead use "present" and "missing" as the terminology to describe if an API exists in workers or not.

Reply:
Yeah, I saw the wording changes you all suggested in the doc after pushing this - can change